ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -115,9 +115,10 @@ void NGraphEncapsulateOp::CreateParallelExecutor(OpKernelConstruction* ctx,
 
   int cluster_id{-1};
   OP_REQUIRES_OK(ctx, ctx->GetAttr<int>("ngraph_cluster", &cluster_id));
-  graph_def = NGraphClusterManager::GetClusterGraph(cluster_id);
+  OP_REQUIRES_OK(ctx, NGraphClusterManager::GetClusterGraph(
+                          cluster_id, encap_subgraph.get()));
 
-  if (graph_def == nullptr) {
+  if (encap_subgraph.get() == nullptr) {
```

Comment:
If GetClusterGraph returns a status, then this check can go away. We can still keep it around for "onion defense" :D